src/PluginManager.js

```diff
@@ -76,6 +76,15 @@ export const regexRules = [
         note: "The full-width characters '{matchString}' are not allowed in the target string. Use ASCII symbols instead.",
         regexps: [ "[\\uFF01|\\uFF05|\\uFF1F]+" ],
         link: "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-no-fullwidth-punctuation-subset.md"
+    },
+        {
+        type: "resource-target",
+        name: "resource-no-space-between-double-byte-kana-and-single-byte-character",
+        description: "Ensure that the target does not contain a space character between a double-byte kana and single-byte character.",
+        note: "The space character is not allowed in the target string. Remove the space character.",
+        regexps: [ "([\\u3040-\\u309F]|[\\u30A0-\\u30FF]|[\\u4E00-\\u9FAF])\\s+[\\x00-\\xFF]|[\\x00-\\xFF]\\s+([\\u3040-\\u309F]|[\\u30A0-\\u30FF]|[\\u4E00-\\u9FAF])" ],
```

Comment:
These are the right reg exps to get all the kanji as well as the kana. It can be a little more efficient by not doing an "or" and just making one big character class:

`([\\u3040-\\u309F]|[\\u30A0-\\u30FF]|[\\u4E00-\\u9FAF])` is the same as
`[\\u3040-\\u309F\\u30A0-\\u30FF\\u4E00-\\u9FAF]`

Reply:
Fixed.